ext/date/php_date.c

```diff
@@ -2832,6 +2837,34 @@ PHP_METHOD(DateTime, __wakeup)
 }
 /* }}} */
 
+/* {{{ proto DateTime::createFromImmutable(DateTimeImmutable object)
+   Creates new DateTime object from an existing DateTimeImmutable object.
+*/
```

Comment:
I was just following the file's current style, which has the ending comment marker on a new line. Examples:

- https://github.com/Rican7/php-src/blob/429b0bb38a32b85e2f56d7669724f1c5aa902baa/ext/date/php_date.c#L2592-L2594
- https://github.com/Rican7/php-src/blob/429b0bb38a32b85e2f56d7669724f1c5aa902baa/ext/date/php_date.c#L3741-L3743
- https://github.com/Rican7/php-src/blob/429b0bb38a32b85e2f56d7669724f1c5aa902baa/ext/date/php_date.c#L4089-L4091